extensions/amp-scroll-toggle/validator-amp-scroll-toggle.protoascii

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-scroll-toggle
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-scroll-toggle"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-scroll-toggle>
+  html_format: AMP
+  tag_name: "AMP-SCROLL-TOGGLE"
+  requires_extension: "amp-scroll-toggle"
+  attrs: {
+    name: "position"
+    value_casei: "bottom"
+    value_casei: "top"
+  }
+  attrs: {
+    name: "target"
+    mandatory: true
+    value_regex: "^[A-Za-z]+[\\w\\-\\:\\.]*$"
```

Comment:
this isn't necessary since `id` implements it, but since `target` here is for an `id` consider also using the blacklist for `id` here as well.

Reply:
Duplicated blacklist for now, which is less than ideal. We wouldn't need it if we go for the attribute approach (@aghassemi to weigh in).

Reply:
(Going for attribute approach, these lists will be removed soon).